project/SparkRedshiftBuild.scala

```diff
@@ -57,6 +57,7 @@ object SparkRedshiftBuild extends Build {
       libraryDependencies ++= Seq(
         "com.amazonaws" % "aws-java-sdk-core" % "1.9.40",
         "com.amazonaws" % "aws-java-sdk-s3" % "1.9.40",
+        "com.amazonaws" % "aws-java-sdk-sts" % "1.9.40" % "test",
```

Comment:
It's only needed in the integration tests, where we use these APIs to obtain STS credentials for use in the tests.